### PR TITLE
PUBSTWO-1700 (put the report number in the <publisher_item> tag)

### PR DESCRIPTION
**Pull request by orosz-usgs**
In body.ftlx  template, added <publisher_item> tag and removed the
<component> tag as described in the issue.
  Deleted unused Bean numberedSeriesXml() and corresponding xml resource.
  Deleted unused Bean ofr20131259Xml() and corresponding xml resource.
  Updated test result testOneNumberedSeriesPub.xml with the tags changes.

### CODE REVIEW

**Review by dsteinich — COMMENTED** [review]
Should the header.ftlx be updated to reflect version 4.4.2 also?

**Comment by orosz-usgs** [comment]
Good question, I'll confirm with Jim if we are good to update.

**Comment by orosz-usgs** [comment]
Jim made PUBSTWO-1717 ticket for the crossref schema version update.

File: src/test/resources/testResult/testOneNumberedSeriesPub.xml [hunk 1]
```diff
@@ -24,6 +24,9 @@
                 <publication_date media_type="online">
                     <year>2013</year>
                 </publication_date>
+                <publisher_item xmlns="http://www.crossref.org/schema/4.4.2">
```
Comment (jkreft-usgs) [inline src/test/resources/testResult/testOneNumberedSeriesPub.xml]: Why is there an extra namespace declaration here?  It should be at the top of the document

File: src/main/resources/templates/crossref/body.ftlx [hunk 2]
```diff
@@ -13,6 +13,9 @@
 				<publication_date media_type="online">
 					<year>${pub.publicationYear}</year>
 				</publication_date>
+				<publisher_item xmlns="http://www.crossref.org/schema/4.4.2">
```
Comment (jkreft-usgs) [inline src/main/resources/templates/crossref/body.ftlx]: Why is there an extra namespace declaration here? It should be at the top of the document

Comment (orosz-usgs) [reply src/main/resources/templates/crossref/body.ftlx]: good catch, taken out.

File: src/test/resources/testResult/testOneNumberedSeriesPub.xml (hunk 1 above)
Comment (orosz-usgs) [reply src/test/resources/testResult/testOneNumberedSeriesPub.xml]: good catch, taken out.

**Review by dsteinich — APPROVED** [review]

**Pull request merged**